azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -123,6 +119,8 @@
   private static final int MAX_FORM_CONTENT_SIZE = 10 * 1024 * 1024;
   private static final String DEFAULT_TIMEZONE_ID = "default.timezone.id";
   private static final String DEFAULT_STATIC_DIR = "";
+
+  @Deprecated
```

Comment:
It is currently being accessed in other classes and probably downstream plugins like `TriggerPlugin` for example. It may break something.